javascript-practice/src/js/controllers/dashboardController.js

```diff
@@ -0,0 +1,39 @@
+import UserModel from '../models/userModel';
+import UserService from '../services/userService';
+import DashboardView from '../views/dashboardView';
+
+export default class DashboardController {
+  constructor() {
+    this.view = new DashboardView();
+    this.model = new UserModel();
+    this.service = new UserService();
+  }
+
+  addUser = async (
```

Comment:
A good function should NOT have more than 3 parameters, pls consider another way

Reply:
I'll update later

Reply:
Got it